processor/src/main/java/org/mapstruct/ap/internal/model/common/Parameter.java

```diff
@@ -119,12 +126,18 @@ public boolean equals(Object o) {
     }
 
     public static Parameter forElementAndType(VariableElement element, Type parameterType) {
```

Comment:
This method is not used anymore. I think that there was a single usage of it and now that usage uses the overloaded method with `isVarArgs`